utils/build-script-impl

```diff
@@ -2361,6 +2360,13 @@ for host in "${ALL_HOSTS[@]}"; do
                     )
                 fi
 
+                if [[ "${SWIFT_INSTALL_COMPONENTS}" ]] ; then
```

Comment:
Ok. I am fine with this. I prefer the -z -n slightly since it makes things more explicit. But if this works... +1!

Everything else looks fine to me!